ambry-store/src/test/java/com/github/ambry/store/BlobStoreStatsTest.java

```diff
@@ -90,7 +90,7 @@ public BlobStoreStatsTest(boolean isBucketingEnabled) throws IOException, StoreE
   private BlobStoreStats setupBlobStoreStats(int bucketCount, long logSegmentForecastOffsetMs) {
     return new BlobStoreStats("", state.index, bucketCount, BUCKET_SPAN_IN_MS, logSegmentForecastOffsetMs,
         QUEUE_PROCESSOR_PERIOD_IN_Ms, DEFAULT_WAIT_TIMEOUT_SECS, true, true, state.time, indexScannerScheduler,
-        queueProcessorScheduler, diskIOScheduler, METRICS);
+        queueProcessorScheduler, diskIOScheduler, METRICS, 1, false);
```

Comment:
Could we leave a TODO saying that currently the valid data size background job is not tested and needs changes to the logic to check the number of throttling events?

I think this is just a test issue that needs to be fixed. I think we equate calls to throttle as a sign that the indexscanner is running and has passed over a certain number of entries. But since getValidSize also iterates over entries it also adds to the throttle count. For now, we can ignore it and leave the TODO. The test may need many changes since IndexScanner is not the only thing scanning the index now.

Reply:
Sure, added the TODO on top of setupBlobStoreStats since many tests need to be fixed later.